src/core/blockchain/types/config.ts

```diff
@@ -11,6 +11,7 @@ export interface IBlockchainConfig {
     derivationType: DerivationType;
     coin: string;
     defaultUnit: string;
+    iconComponent: any;
```

Comment:
use `React.ComponentType` instead of any

Reply:
`React.ComponentType` seems not to be working:

![image](https://user-images.githubusercontent.com/32679726/76197285-833dc780-61f4-11ea-8bbf-4f5e7c892581.png)